db_api.rkt

```diff
@@ -115,9 +118,9 @@
                         (view-conn-info v)
                         (λ (c) (query-exec c (insert-query-string v all-cols all-values))))))))))
 
-(interface view-iface (get-struct where select join fetch update delete insert))
+(interface dbview (where select join fetch update delete insert))
```

Comment:
Renamed to `dbview` so it will look nicer from shill. Ideally would change to `view`, but that requires renaming all the references to the view struct in `utils.rkt` (or at least changing the import to `(prefix-in utils: "utils.rkt")`.